--- a/adm-zip.js	
+++ b/adm-zip.js	
@@ -539,9 +539,9 @@
                 } else {
                     filetools.findFilesAsync(localPath, function (err, fileEntries) {
                         if (err) return callback(err);
                         fileEntries = fileEntries.filter((dir) => filter(relPathFix(dir)));
-                        if (!fileEntries.length) callback(undefined, false);
+                        if (!fileEntries.length) callback(undefined, true);
 
                         setImmediate(
                             fileEntries.reverse().reduce(function (next, entry) {
                                 return function (err, done) {
